packages/experience-builder-sdk/src/ExperienceRoot.tsx

```diff
@@ -38,6 +38,7 @@ export const ExperienceRoot = ({
         experience={experienceObject as Experience<EntityStore> | undefined}
         visualEditorMode={visualEditorMode}
         initialLocale={locale}
+        isReadOnlyMode={isReadOnlyMode}
       />
```

Comment:
For read only we still render the VisualEditorRoot, but we set it into read only